ext/lmdb-safe/lmdb-safe.hh

```diff
@@ -12,9 +15,95 @@
 #include <mutex>
 #include <vector>
 #include <algorithm>
+#include <bit>
+#include <byteswap.h>
```

Comment:
on macOS:

```
../../ext/lmdb-safe/lmdb-safe.hh:19:10: fatal error: 'byteswap.h' file not found
```

Reply:
As a workaround I commented the include out and added this from the header file (renamed):

```cpp
#define bswap_64(x)			\
  ((((x) & 0xff00000000000000ull) >> 56)	\
   | (((x) & 0x00ff000000000000ull) >> 40)	\
   | (((x) & 0x0000ff0000000000ull) >> 24)	\
   | (((x) & 0x000000ff00000000ull) >> 8)	\
   | (((x) & 0x00000000ff000000ull) << 8)	\
   | (((x) & 0x0000000000ff0000ull) << 24)	\
   | (((x) & 0x000000000000ff00ull) << 40)	\
   | (((x) & 0x00000000000000ffull) << 56))
```

Reply:
I switched it to `__builtin_bswap64` which is available in GCC and Clang.